Changelog.md

```diff
@@ -8,6 +8,7 @@ Compiler Features:
 
 Bugfixes:
  * Type Checker: Disallow ``virtual`` and ``override`` for constructors.
+ * Standard Json Input: Fix error when using prefix ``file://`` in the field ``urls``.
```

Comment:
I wouldn't say this is a bugfix, nor it is a bugfix for "standard json". It is a feature for the CLI.

Reply:
According to the https://github.com/ethereum/solidity/issues/4884 it's a bug. And I agree with it because if it don't work as documented it looks like a bug, IMHO

Reply:
Well, it's either a bugfix in the code or in the documentation, since those were inconsistent. But I'm fine with either